lib/configuration.py

```diff
@@ -0,0 +1,130 @@
+import logging
```

Comment:
Config parsing logic copy/pasted from the original `peloton-to-garmin.py` file.  Moved here for better single responsibility of code and testability.